applications/PoromechanicsApplication/python_scripts/poromechanics_MPI_U_Pw_solver.py

```diff
@@ -224,9 +184,9 @@ def _ConstructSolver(self, builder_and_solver, scheme, convergence_criterion, st
         if strategy_type == "Newton-Raphson":
```

Comment:
not for this PR but according to the [Style Guide](https://github.com/KratosMultiphysics/Kratos/wiki/Style-Guide#json-configuration-file) the name should be `newton_raphson` => also at other places